src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -164,6 +185,26 @@ s! {
         __f_spare: [::c_int; 6],
     }
 
+    pub struct statvfs64 {
```

Comment:
Could you add a comment to `statvfs64` and `statfs64` that we always assume largefile64? Since these types seem to be gated by `_LARGEFILE64_SOURCE`.